recipes/sphinx-jinja/LICENSE.txt

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
Please make a pr upstream for them to include this license in their manifests.

Reply:
1.1.1 was just published with License details in it..